### PR TITLE
Hero update

### LINKED ISSUE #2: Finalise Cycle7 logo and add to site

### PR DESCRIPTION
**Pull request by patrickbower**
Add logo lockup and dummy image

closes #2

### CODE REVIEW

File: src/styles/hero.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.hero-logo-lockup {
+  width: 400px;
```
Comment (danbull) [inline src/styles/hero.scss]: For smaller screens it's probably best to make this a `max-width` for specificity I think you'll have to change the selector to `.grav-c-hero .grav-c-hero__container .hero-logo-lockup`

File: src/styles/hero.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  fill: var(--grav-co-grp-b-neutral);
+}
+
+.hero-img {
+  background: url(../images/hero-test.jpg);
+  background-size: cover;
+  min-height: 450px;
+  width: 100%;
+}
```
Comment (danbull) [inline src/styles/hero.scss]: We could use an inline image for this like the Gravity homepage hero pattern? http://style-next.buildit.digital/components/detail/page-homepage.html

Comment (patrickbower) [reply src/styles/hero.scss]: oh yeah, not seen the `object-fit` css prop before - nice

File: src/components/hero.js [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+  </g>
+</svg>
+      {/* <h1>Cycle 7</h1> */}
```
Comment (danbull) [inline src/components/hero.js]: We have to have an `h1` so maybe let's go with Cycle 7 and make it SR only

**Pull request merged**